### PR TITLE
Fix nested modifiers' arity

### LINKED ISSUE #1252: Nested modifiers don't retain arity
[Try it Online!](https://vyxal.pythonanywhere.com/#WyIiLCIiLCJ2disiLCIiLCJbMSwyLDNdXG5bWzEsMl0sWzQsNV1dIl0=)

[Try it Online!](https://vyxal.pythonanywhere.com/#WyIiLCIiLCJ2zrsyfHYrIiwiIiwiWzEsMiwzXVxuW1sxLDJdLFs0LDVdXSJd)

[Try it Online!](https://vyxal.pythonanywhere.com/#WyIiLCIiLCLigo0r4oKNLSoiLCIiLCIyXG4zIl0=)

[Try it Online!](https://vyxal.pythonanywhere.com/#WyIiLCIiLCLigo0rzrsyfOKCjS0qIiwiIiwiMlxuMyJd)

Of course the `2` flag fixes it, but that can mess up other stuff.

### PR DESCRIPTION
**Pull request by Steffan153**
Closes #1252

### CODE REVIEW

File: vyxal/transpile.py [hunk 1]
```diff
@@ -4,6 +4,8 @@
 import secrets
 from typing import Union
 
+from sympy import arity
```
Comment (chunkybanana) [inline vyxal/transpile.py:7]: Where is this used?

Comment (Steffan153) [reply vyxal/transpile.py:7]: Some autoimport VScode did, I had no idea about that.

```suggestion
```

File: vyxal/transpile.py [hunk 2]
```diff
@@ -4,6 +4,7 @@
 import secrets
 from typing import Union
 
+
```
Comment (Steffan153) [inline vyxal/transpile.py:7]:
```suggestion
```

**Review by chunkybanana — APPROVED** [review]

**Comment by chunkybanana** [comment]
This might also fix #1195

**Review by Lyxal — APPROVED** [review]

**Pull request merged**